client/components/domains/domain-product-price/index.jsx

```diff
@@ -15,12 +18,17 @@ const DomainProductPrice = React.createClass( {
 		requiresPlan: React.PropTypes.bool
 	},
 	renderFreeWithPlan() {
+		const product_price = ! domainsWithPlansOnlyTestEnabled &&
```

Comment:
Ha-ha! The tables have turned! This time it's _me_ telling _you_ that in _JavaScript_, we use camelCase for variable names, not snake_case like in PHP 😜 :wink: